### PR TITLE
Redshift profile iam fix

### PR DESCRIPTION
**Pull request by Fleid**

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/redshift-setup.md [hunk 1]
```diff
@@ -43,20 +43,19 @@ import SetUpPages from '/snippets/_setup-pages-intro.md';
...
 
-The authentication methods that dbt Core supports are: 
+The authentication methods that dbt supports on Redshift are: 
 
-- `database` &mdash; Password-based authentication (default, will be used if `method` is not provided)
-- `IAM` &mdash; IAM
-
-For dbt Cloud users, log in using the default **Database username** and **password**. This is necessary because dbt Cloud does not support `IAM` authentication.
+- `Database` &mdash; Password-based authentication (default, will be used if `method` is not provided)
+- `IAM User` &mdash; IAM User authentication, via AWS Profile on **Core only** and inline on **Cloud only**
```
Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/redshift-setup.md]:
```suggestion
- `IAM User` &mdash; IAM User authentication via AWS Profile on _Core only_ and inline on _Cloud only_
```

File: website/docs/docs/core/connect-data-platform/redshift-setup.md [hunk 2]
```diff
@@ -43,20 +43,19 @@ import SetUpPages from '/snippets/_setup-pages-intro.md';
...
 
 Click on one of these authentication methods for further details on how to configure your connection profile. Each tab also includes an example `profiles.yml` configuration file for you to review.
 
 <Tabs
   defaultValue="database"
   values={[
-    {label: 'database', value: 'database'},
-    {label: 'IAM', value: 'IAM'},
+    {label: 'Database (both)', value: 'database'},
+    {label: 'IAM User via AWS Profile (core)', value: 'iam-user-profile'},
```
Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/redshift-setup.md]:
```suggestion
    {label: 'IAM User via AWS Profile (Core)', value: 'iam-user-profile'},
```

File: website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-connection.png" width="70%" title="Configuring a Redshift connection"/>
 
-For dbt Cloud users, please log in using the default Database username and password. Note this is because [`IAM` authentication](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-user-credentials.html) is not compatible with dbt Cloud.
+For **authentication**, dbt Cloud users can pick Database username and password. In addition, it is now possible to use IAM User authentication to Redshift, via [extended attributes](/docs/core/connect-data-platform/redshift-setup#authentication-parameters).
```
Comment (matthewshaver) [inline website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md]:
```suggestion
For authentication, dbt Cloud users can use either a Database username and password, or they can now use IAM User authentication to Redshift via [extended attributes](/docs/core/connect-data-platform/redshift-setup#authentication-parameters).
```

File: website/docs/docs/core/connect-data-platform/redshift-setup.md [hunk 4]
```diff
@@ -168,14 +165,59 @@ please refer to the official AWS documentation on [Configuration and credential
...
+#### Specifying an IAM Profile
+
+When the `iam_profile` configuration is set, dbt will use the specified profile from your `~/.aws/config` file instead of using the profile name `default`
+
 </TabItem>
 
-</Tabs>
+<TabItem value="iam-user-inline">
+
+The following table lists the authentication parameters to use IAM authentication **on dbt Cloud**.
```
Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/redshift-setup.md:176]: This is the Core setup guide, should this be
```suggestion
The following table lists the authentication parameters to use IAM authentication **on dbt Core**.
```

Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/redshift-setup.md]: We don't generally have the Cloud instructions on the Core adapter pages. My concern is that a user will search, end up here, and maybe not realize the rest is Core related only or become confused. Would it be enough to only include these on the Cloud adapter setup page?

Comment (Fleid) [reply website/docs/docs/core/connect-data-platform/redshift-setup.md]: Moved everything in the following commit, thanks Matt, good call :)

File: website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md [hunk 5]
```diff
@@ -17,7 +17,75 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
...
-For dbt Cloud users, please log in using the default Database username and password. Note this is because [`IAM` authentication](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-user-credentials.html) is not compatible with dbt Cloud.
+### Authentication Parameters
+
+For authentication, dbt Cloud users can use either a **Database username and password**, or they can now use **IAM User authentication** to Redshift via [extended attributes](/docs/dbt-cloud-environments#extended-attributes).
+
+<Tabs
+  defaultValue="database"
+  values={[
+    {label: 'Database', value: 'database'},
+    {label: 'IAM User', value: 'iam-user-inline'}
```
Comment (matthewshaver) [inline website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md]:
```suggestion
    {label: 'IAM User', value: 'iam-user-inline'},
```

File: website/docs/docs/core/connect-data-platform/redshift-setup.md [hunk 6]
```diff
@@ -43,20 +43,18 @@ import SetUpPages from '/snippets/_setup-pages-intro.md';
...
 
 Click on one of these authentication methods for further details on how to configure your connection profile. Each tab also includes an example `profiles.yml` configuration file for you to review.
 
 <Tabs
   defaultValue="database"
   values={[
-    {label: 'database', value: 'database'},
-    {label: 'IAM', value: 'IAM'},
+    {label: 'Database', value: 'database'},
+    {label: 'IAM User via AWS Profile (Core)', value: 'iam-user-profile'}
```
Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/redshift-setup.md]:
```suggestion
    {label: 'IAM User via AWS Profile (Core)', value: 'iam-user-profile'},
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**